main_2023.py

```diff
@@ -1,30 +1,40 @@
 """
 For 2022-2023 UAS competition.
 """
+import argparse
 import multiprocessing as mp
 
 import cv2
+import yaml
 
 from utilities.workers import queue_proxy_wrapper
 from utilities.workers import worker_controller
 from utilities.workers import worker_manager
 from modules.detect_target import detect_target_worker
 from modules.video_input import video_input_worker
 
+if __name__ == "__main__":
 
-QUEUE_MAX_SIZE = 10
+    # open command file
+    with open("config.yaml", 'r', encoding="utf8") as file:
```

Comment:
Is there a possibility of this failing? What happens if the file doesn't exist? How is that handled?